### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by nschejtman**
- W-11267027: fix operation.withResponse(name) interface
- Adopt GovernanceMode on adhoc-cli

### CODE REVIEW

File: adhoc-cli/src/main/scala/amf/adhoc/cli/Main.scala [hunk 1]
```diff
@@ -75,9 +75,9 @@ object Main {
     baseUnit = transformation.baseUnit
 
     val renderOptions = if (withLexical) {
-      RenderOptions().withPrettyPrint.withSourceMaps.withSourceInformation
+      RenderOptions().withPrettyPrint.withGovernanceMode.withSourceMaps.withSourceInformation
```
Comment (hghianni) [inline adhoc-cli/src/main/scala/amf/adhoc/cli/Main.scala]: The withSourceMaps is unnecessary here

Comment (nschejtman) [reply adhoc-cli/src/main/scala/amf/adhoc/cli/Main.scala]: The lexical information is included in the `.withSourceMaps` or `.withSourceInformation` render option?

Comment (hghianni) [reply adhoc-cli/src/main/scala/amf/adhoc/cli/Main.scala]: I understand that lexical is included with `.withSourceMap` but the `GovernanceMode` will filter them.

Comment (hghianni) [reply adhoc-cli/src/main/scala/amf/adhoc/cli/Main.scala]: So `.withSourceMaps` should be used instead of `withGovernanceMode`

**Review by hghianni — DISMISSED** [review]

**Review by hghianni — APPROVED** [review]

**Pull request merged**